src/app.ts

```diff
@@ -15,6 +23,60 @@ const findBranchesToProcess = () => {
   return new RegExp(branches)
 }
 
+const processPullRequest = () => async (context: Context<'pull_request'>, app: Probot) => {
```

Comment:
We can avoid passing in the entire `Probot` object in function calls by just taking the Pino `Logger`. That'll also follow the logic in the rest of the app implementation for consistency 😄 

```suggestion
const processPullRequest = (log: Logger) => async (context: Context<'pull_request'>) => {
```

Reply:
Ah, yeah - that'd be optimal 😄